### ISSUE TITLE
TuMangaOnline: HTTP error 499/500

### ISSUE THREAD

**Issue by Vladraig** [tachiyomiorg/tachiyomi-extensions#15174]
### Source information and language

TuMangaOnline 1.3.39 Spanish

### Steps to reproduce

Http 499
Http 500
but only once in a while

### Expected behavior

should update the recent chapters

### Actual behavior

errors 499 and 500 appear

### Tachiyomi version

0.14.3

### Android version

Android 8.1.0

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by alessandrojean** [comment]
These are server side errors and we can't do anything about it. As you said it happens once in a while, it's probably in moments their servers are facing issues.
